src/getDatasetHelpers.js

```diff
@@ -29,6 +30,11 @@ const splitPrefixIntoParts = (prefix) => {
     .replace(/\/$/, '')
     .split("/");
 
+  /* /groups is backed by a meta Source */
```

Comment:
An alternative approach to modifying these functions would be to modify the `getAvailable` handler and special case this API request, e.g.

```js
  if (prefix==="/groups") {
    const source = metaSources.get("groups");
    return res.json({
      datasets: await source.availableDatasets(),
      narratives: await source.availableNarratives()
    });
  }
```

Reply:
(I've updated this PR to use this approach)